src/block/progress-bar/index.php

```diff
@@ -19,3 +19,14 @@ function stackable_load_progressbar_frontend_script() {
 	}
 	add_action( 'stackable/progress-bar/enqueue_scripts', 'stackable_load_progressbar_frontend_script' );
 }
+
+// Remove commas when the progress value is text field from dynamic content
+if ( ! function_exists( 'stackable_progress_bar_value_remove_commas' ) ) {
+	function stackable_progress_bar_value_remove_commas( $block_content, $block ) {
+		return preg_replace_callback( '/--progress-value:[^%]+%/', function ( $matches ) {
```

Comment:
Please add a check prior to this whether `$block_content` is not null, and also only proceed if the `$block-content` string includes a comma `,` (do this in the other block too)